src/crystal/init_project/template/gitignore.ecr

```diff
@@ -0,0 +1,4 @@
+/.deps/
+/.deps.lock
```

Comment:
If you generate gem with `bundle gem`, then `Gemfile.lock` is in `.gitignore`.

This is because for libraries it is OK to break on newest versions (and you want to know about that, for example on travis build). Because they will be locked in applications that use them as a dependency.

On the other hand, bundler and rubygems do have gemspec (which actually is used for locking down versions, at least constraining them) - which is committed to source control.

Applications on the other hand will definitely want to lock down the versions and commit `.deps.lock`.